source/common/network/udp_listener_impl.cc

```diff
@@ -104,6 +113,9 @@ void UdpListenerImpl::handleReadCallback() {
 
     Address::InstanceConstSharedPtr local_address = socket_.localAddress();
 
+    RELEASE_ASSERT((local_address != nullptr),
```

Comment:
Can this actually happen?

Reply:
I haven't dug deep into that. I refactored existing code which was doing this check